### DISCUSSION TITLE
Reverse fill mask option with maximum length ?

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#9425] [General - Components / Directives / etc] [answered]
When reverse fill mask option added to input field with mask, it makes non-fixed length the input. Is it possible to use mask decimals with reverse fill mask option and a fixed length? Thank you.

**Comment by ontwikkelfabriek** [comment, marked as answer]
@mecjos 

If you want better support please consider creating a codepen(very easy to do) to demonstrate your problem.
https://codepen.io/rstoenescu/pen/VgQbdx

**Reply by mecjos** [reply]
https://codepen.io/pen/?&editors=101

It's same with doc's example actually.. I want to limit max digit like in mysql db input decimal(6,2).. When you add reverse-fill-mask option, you can write unlimited digits. Briefly I want to combine reverse-fill-mask and max-digit.

**Reply by ontwikkelfabriek** [reply]
@mecjos 

Check you codepen it is not saved and empty

**Reply by mecjos** [reply]
Please check the link below.. It's same with that title's codepen..

https://quasar.dev/vue-components/input#example--filling-the-mask-in-reverse

**Reply by mecjos** [reply]
I want to add max digit limitation on it.

**Reply by ontwikkelfabriek** [reply]
@mecjos 
Ah I see the problem...

Here's an attempt to set the maxlength:
https://codepen.io/ontwikkelfabriek/pen/poeWmXQ

**Reply by mecjos** [reply]
Oh it was so simple :) I couldn't find it in doc.. Thank you.. )

**Reply by ontwikkelfabriek** [reply]
Here's the doc section:
https://quasar.dev/vue-components/input#native-attributes
